### ISSUE TITLE
I got an uncountable amount of errors when running 'yo code'

### ISSUE THREAD

**Issue by BlueStaggo** [yeoman/yo#664]
## Type of issue

Bug

----

## My environment

* *OS version/details*: `Windows 10 x64 1909`
* *Node version:* `v16.0.0`
* *npm version:* `7.8.0`
* *Version of yo :* I don't know because I can't run `yo --version` (that throws a ton of errors as well). It's probably `4.0.0`

## Expected behavior

Something like this:
```
yo code

# ? What type of extension do you want to create? New Extension (TypeScript)
# ? What's the name of your extension? HelloWorld
### Press <Enter> to choose default for all options below ###

# ? What's the identifier of your extension? helloworld
# ? What's the description of your extension? LEAVE BLANK
# ? Initialize a git repository? Yes
# ? Bundle the source code with webpack? No
# ? Which package manager to use? npm

code ./helloworld
```

## Current behavior

Whatever this is:
![image](https://user-images.githubusercontent.com/71090710/116440315-4691bd00-a848-11eb-8661-eaba4cefdba4.png)
I zoomed out Visual Studio Code to the minimum and I can only see a small amount of the errors! This is Powershell, by the way.

## Steps to reproduce the behavior

1. Install yo and generator-code.
2. Type in `yo code` or any other yo command.

## Command line output

See current behavior.

## Notes
* You don't need generator-code for this.
* This only happens in the latest release. I installed 3.1.0 and it worked.

**Comment by RedCuckoo** [comment]
I encounter the same problem, thank you for workaround with installing not the latest version.

My environment, where it is reproducible:

- OS: Linux Mint (Ubuntu) with KDE

- Node version: v16.0.0

- NPM version: 7.10.0

- yo version: unknown (yo --version produces this errors)

It's happening after doing `npm install -g yo` or `npm install -g yo@latest`

**Comment by mshima** [comment]
Seems duplicate of https://github.com/yeoman/yo/issues/662

**Comment by mshima** [comment]
Should be fixed in 4.1.0
